scripts/genAbi.js

```diff
@@ -42,6 +43,8 @@ async function main() {
   const allFiles = glob(cwd, [
     `${peripheralsPath}/build/contracts/!(build-info)/**/+([a-zA-Z0-9_]).json`,
     `${nitroPath}/build/contracts/!(build-info)/**/+([a-zA-Z0-9_]).json`,
+    // we have a hardcoded abi for the old outbox
+    `./src/lib/dataEntities/OldOutbox.json`,
```

Comment:
I believe we want to handle 2 diff "old outboxes"
[arb classic Outbox](https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-eth/contracts/bridge/Outbox.sol) and [arb classic OldOutbox](https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-eth/contracts/bridge/Old_Outbox/OldOutbox.sol) / outbox entries

Reply:
Added support for both